modules/log6x-6-1-2-rn.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.1/log6x-release-notes-6.1.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-1-2_{context}"]
+= Logging 6.1.2 Release Notes
+
+This release includes link:https://access.redhat.com/errata/RHBA-2025:1229[{logging-uc} {for} Bug Fix Release 6.1.2].
+
+[id="logging-release-notes-6-1-2-enhancements_{context}"]
+== New Features and Enhancements
+
+* This enhancement adds `OTel` semantic stream labels to the `lokiStack` output so that you can query logs by using both `ViaQ` and `OTel` stream labels.
+(link:https://issues.redhat.com/browse/LOG-6579[LOG-6579])
+
+[id="logging-release-notes-6-1-2-bug-fixes_{context}"]
+== Bug Fixes
+
+* Before this update, the collector alerting rules contained summary and message fields. With this update, the collector alerting rules contain summary and description fields.
+(link:https://issues.redhat.com/browse/LOG-6126[LOG-6126])
+
+* Before this update, you could remove the collector metrics dashboard after an operator upgrade due to a race condition during the transition from the old to the new pod deployment. With this update, you add labels to the dashboard `ConfigMap` to identify the upgraded deployment as the current owner, preventing you from removing the collector metrics dashboard after an operator upgrade.
+(link:https://issues.redhat.com/browse/LOG-6280[LOG-6280])
+
+* Before this update, when you included infrastructure namespaces in application inputs, the `log_type` was set as `application`. With this update, you set the `log_type` of infrastructure namespaces included in application inputs to `infrastructure`.
```

Comment:
```suggestion
* Before this update, when you included infrastructure namespaces in application inputs, the `log_type` parameter was set as `application`. With this update, you set `log_type` of infrastructure namespaces included in application inputs to `infrastructure`.
```

- Please use field instead of parameter is it's more suitable

Specify the object name as per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references